Makefile

```diff
@@ -218,6 +218,7 @@ dev: ## Set Environment to DEV
 	$(eval export SLIM_DEV_DEPLOYMENT ?= true)
 	$(eval export CA_ROTATION_EXPIRY_DAYS ?= 360)
 	$(eval export ENABLE_PAAS_ADMIN_CONTINUOUS_DEPLOY ?= true)
+	$(eval export DISABLED_AZS)
```

Comment:
How should this be used at the command line, given the input is space separated? As below?

```
make ENV pipelines DISABLED_AZS="z1 z2"
```

Reply:
Bingo.

Reply:
Confirmed it worked for my own sanity